converters/toZigbee.js

```diff
@@ -660,15 +660,27 @@ const converters = {
         key: ['weekly_schedule'],
         convertSet: async (entity, key, value, meta) => {
             const payload = {
-                temperature_setpoint_hold: value.temperature_setpoint_hold,
```

Comment:
As far as I can see code that was here before cannot work because payload format doesn't match setWeeklySchedule command (from the name it looks like it was hvacThermostat attributes instead).